tutorials/kafka-live-viewer-profiles/conclusion.md

```diff
@@ -3,39 +3,39 @@ position: 4
 title: Conclusion
 ---
 
-# Conclusion
+In this tutorial, you've explored the **live viewer profiles** solution accelerator for video streaming, gaining practical insights into building, deploying, and extending real-time, event-driven architectures using Snowplow and Kafka.
 
-In this tutorial, we have explored the **live viewer profiles** solution accelerator for video streaming, gaining practical insights into building, deploying, and extending real-time, event-driven architectures using Snowplow and Kafka.
+## Understanding the process
 
-## Key Takeaways
-
-### Understanding the Process
-We successfully have built a real time system for processing event data including:
-- **Web Tracking Application** for collecting media events.
+You have successfully built a real time system for processing event data including:
+- **Web tracking application** for collecting media events
 ![Application Output](images/video.png)
-- **Snowplow Collector and Snowbridge** for event processing and forwarding.
-- **Live Viewer Backend** for managing real-time data with Kafka and DynamoDB.
-- **Live Viewer Frontend** for visualizing real-time user activity on the web tracking application.
+- **Snowplow Collector and Snowbridge** for event processing and forwarding
+- **Live Viewer back-end** for managing real-time data with Kafka and DynamoDB
+- **Live Viewer front-end** for visualizing real-time user activity on the web tracking application
 ![Live viewer frontend](images/live-viewer.png)
 
 This architecture highlights how real-time insights can be achieved using event-driven systems in a streaming context.
 
-### What We Achieved
-We explored how to:
-1. Use LocalStack to emulate AWS services for local development and testing.
-2. Launch and interact with the system components, such as the Kafka UI and LocalStack UI.
-3. View and verify the real-time event data from the browser using Snowplow's media tracking capabilities.
-4. Deploy the solution in an AWS environment using Terraform. 
+## What you achieved
+
+You explored how to:
+1. Use LocalStack to emulate AWS services for local development and testing
+2. Launch and interact with the system components, such as the Kafka UI and LocalStack UI
```

Comment:
Haven't run the accelerator but does Localstack provide a UI in the free version?